content/en/real_user_monitoring/guide/understanding-the-rum-event-hierarchy.md

```diff
@@ -0,0 +1,114 @@
+
+{{< img src="real_user_monitoring/guide/understanding-rum-event-hierarchy/rum-switch-views.mp4" alt="RUM views" video="true" style="width:80%;">}}
+
+Similarly to the `session.id`, each view automatically has a unique `view.id` connected to it. 
+
+### Actions, errors, resources, and long tasks
+
+Within views, the SDK creates more granular events that all fall along the same level hierarchically. However, each event is unique and carries its own attributes and properties.
+
+### Actions
+
+Actions represent user activity on a page. In browsers, all click actions are automatically collected. On mobile, all taps, swipes, and scrolls are collected. Beyond these default actions, you can also send [custom actions][5], such as form completion and business transactions. 
+
+#### Sample search: Top list of all "Add to cart" actions that led to an error
+
+In this example, search for all actions from users clicking on the "Add to cart" button that resulted in an error.
+
+{{< img src="real_user_monitoring/guide/understanding-rum-event-hierarchy/rum-actions-all-add-to-cart.png" alt="Sample search of all "Add to Cart" actions that led to an error." style="width:80%;">}}
+
+### Errors
+
+You can use RUM to collect [frontend errors][6] that occur during the user session. By default, the browser SDK creates error events for unhandled exceptions and console errors. Additionally, you can collect custom errors through the RUM `addError` API ([browser][7] | [mobile][8]). On mobile apps, you can also see if the error led to a session termination, also known as a crash.
+
+Errors can be viewed across both RUM and Error Tracking. Source and custom errors will be processed by Error Tracking, while console errors will be solely in RUM.
+
+#### Sample search: List of all crashes that occurred on the "CheckoutViewController"page in the iOS Shop.ist application
+
+In this example, search within the errors event to see all crashes that occurred on the “CheckoutViewController” page for a particular operating system and application.
+
+{{< img src="real_user_monitoring/guide/understanding-rum-event-hierarchy/rum-sample-search-checkoutviewcontroller.png" alt="Sample search of all crashes that occurred on a page." style="width:80%;">}}
+
+### Resources
+Resources are collected from views and include external requests from your application to a network provider, like XHR, JS loading, images, fonts, etc. Since it is collected from a view, you can query for all resources loaded on an application, or scope it down to just resources that occurred in a single view. 
```

Comment:
```suggestion
Resources are collected from views and include external requests from your application to a network provider, things like XHR, JS loading, images, or fonts, for example. Since it is collected from a view, you can query for all resources loaded on an application, or scope it down to just resources that occurred in a single view. 
```